bigbluebutton-html5/imports/ui/core/hooks/useMeeting.ts

```
@@ -1,18 +1,96 @@
+  Omit<MeetingStaticData, Overlap> & {
+    [K in Overlap]: Meeting[K] | MeetingStaticData[K];
+  }
+>;
+
+// For projection results: allow *top-level* props to be partial if they are objects.
+// (We stay shallow for the public type.)
+type DistributePartial<T> = T extends object ? Partial<T> : T;
+type Loose<T> = { [K in keyof T]?: DistributePartial<T[K]> };
+
+// subscription remains on Meeting
+const useMeetingSubscription = useCreateUseSubscription<Meeting>(
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)